e2e/playwright/utils.ts

```diff
@@ -1,28 +1,41 @@
 import { sign } from "jsonwebtoken";
-import Axios from "axios";
+import { GraphQLClient } from "graphql-request";
 
-export const gqlAdmin = async (query, variables = {}) => {
-  const HASURA_GRAPHQL_ADMIN_SECRET = process.env.HASURA_GRAPHQL_ADMIN_SECRET;
-  const HASURA_PROXY_PORT = process.env.HASURA_PROXY_PORT;
+const HASURA_GRAPHQL_ADMIN_SECRET = process.env.HASURA_GRAPHQL_ADMIN_SECRET;
+const HASURA_PROXY_PORT = process.env.HASURA_PROXY_PORT;
 
-  const response = await Axios(
-    `http://localhost:${HASURA_PROXY_PORT}/v1/graphql`,
-    {
-      method: "POST",
-      headers: {
-        "X-Hasura-Admin-Secret": HASURA_GRAPHQL_ADMIN_SECRET,
-      },
-      data: { query, variables },
-    }
-  );
-  const { data: json } = response;
+const prodClient = new GraphQLClient(
+  "https://hasura.editor.planx.uk/v1/graphql"
+);
```

Comment:
Since production & staging content are now synced regularly, does Playwright actually need to connect to prod or could it query the same flow on staging? Just thinking it's nice to use our prod admin secret sparingly whenever possible!